.github/dependabot.yaml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
```

```suggestion
    directory: ".github/workflows"
```
Shouldn't this be limited to the `.github/workflows` sub-directory?

---

No, the directory is correct.